sphinx_gallery/gen_gallery.py

```diff
@@ -687,6 +692,150 @@ def write_computation_times(gallery_conf, target_dir, costs):
             fid.write(hline)
 
 
+def write_api_entry_usage(gallery_conf, target_dir):
+    if gallery_conf['backreferences_dir'] is None:
+        return
+
+    backreferences_dir = os.path.join(gallery_conf['src_dir'],
+                                      gallery_conf['backreferences_dir'])
+    example_files = [example for example in os.listdir(backreferences_dir)
+                     if '__' not in example and
+                     ((example.endswith('.examples') and
+                       not os.path.isfile(example + '.new')) or
+                     example.endswith('.examples.new'))]
+
+    def get_entry(entry):
```

Comment:
not sure `get_entry` is very descriptive of what this function actually does (you give it an fname and it removes file extensions from it)